chanzuckerberg/idseq-web
# [Data Discovery] Add all you can see (aka. admin) tab

## Description
* Adds an "All Data" tab to data discovery
* Also changes all code references if `library` to `my_data`
* Add Integration tests to visualizations access control

## Test:
* Open DD as a normal user:
	- verify you have no access to the tab through the header menu
	- verify you cannot access `/all_data`
* Open DD as an admin user
	- verify you can access all data
